introduction.md

```diff
@@ -1,4 +1,8 @@
 Hi, I am krjohnston
 My favorite code school path is javascript
+Deadly skills
+* Javascript
+* PHP
+* MySQL
 
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?